package.json

```diff
@@ -2,19 +2,23 @@
   "name": "multihashes",
   "version": "0.4.13",
   "description": "multihash implementation",
-  "main": "src/index.js",
+  "main": "lib/index.js",
```

Comment:
So this will now point at files that doesn't exists when we checkout the code right? Only after running build? How do we make this easy to work with when you have multiple modules that needs building and npm-linking?

Reply:
I think you will run a watcher (part of AEgir) in the project that you link. Is that correct @Gozala?

Reply:
So in practice, you would run one watcher for builds and one watcher for tests, per each project you link + top project?

I'm pretty sure this is alright, but let's check in with the person who probably links most projects currently, @diasdavid. Does the above flow work for you?

Reply:
> I think you will run a watcher (part of AEgir) in the project that you link. Is that correct @Gozala?

That is at least in my understand on what we agreed to do, except you don't necessary need to run watcher on linked packages that you're not making changes to.